godot-core/src/builtin/callable.rs

```diff
@@ -16,6 +16,12 @@ use crate::obj::{Gd, GodotClass, InstanceId};
 use std::{fmt, ptr};
 use sys::{ffi_methods, GodotFfi};
 
+#[cfg(before_api = "4.3")]
+#[cfg(since_api = "4.2")]
+type GDExtensionCallableCustomInfo = sys::GDExtensionCallableCustomInfo;
+#[cfg(since_api = "4.3")]
+type GDExtensionCallableCustomInfo = sys::GDExtensionCallableCustomInfo2;
```

Comment:
The naming is a bit confusing -- the use sites look like `sys::` was forgotten and the code is referring to `sys::GDExtensionCallableCustomInfo`.

Maybe just like this, to make clear it's not a typedef exported by the GDExtension API itself?
```suggestion
#[cfg(since_api = "4.3")]
type CallableCustomInfo = sys::GDExtensionCallableCustomInfo2;
```